lib/pdf_fill/forms/va281900v2.rb

```diff
@@ -229,6 +229,9 @@ class Va281900v2 < FormBase
           question_suffix: 'A',
           question_text: 'NUMBER OF YEARS OF EDUCATION'
         },
+        'privacyAgreementAccepted' => {
+          key: 'form1[0].#subform[1].IfIDontGiveMyInfo[0]'
```

Comment:
I just want to point out that `IfIDontGiveMyInfo` has the same cadence as `If you wanna be my lover`. Remember I told you that.